src/arrayMethodJoin.js

```diff
@@ -4,8 +4,27 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let str = '';
+    let newSeparator = separator;
+
+    if (this.length === 0) {
+      return '';
```

Comment:
```suggestion
      return str;
```
Можна повертати твій результуючий рядок. Він поки що також пустий